add copy to clipboard behaviour

earnubs opened the pull request:
spec stipulates that a copy-to-clipboard icon exist below cli instructions, part of #655

magdalena-mirowicz commented:
please pause this work for now

magdalena-mirowicz commented:
@earnubs please ignore me, it's ok to work on

bartaz commented on src/common/components/share/index.js:
```diff
@@ -0,0 +1,28 @@
+
+export class CopyToClipboard extends Component {
+  componentDidMount() {
+    new Clipboard(`.${COPY_TO_CLIPBOARD_CLASS}`);
```
Will it work just fine if there are two instances of that on a page?

So I suppose `new Clipboard` would be called 2 times with same class name?

earnubs replied:
Good point, I was looking at the spec and expecting only one of these per page.

earnubs commented:
![image](https://cloud.githubusercontent.com/assets/167440/26075388/fdc68550-39ac-11e7-83be-752bd15356cc.png)

earnubs commented on src/common/components/share/index.js:
```diff
@@ -0,0 +1,50 @@
+  }
+
+  initClipboard() {
+    this.clipboard = new Clipboard(this.copyBtn);
```
Only add the eventListener to a single component instance, so multiple instances of this component can live with each other.

bartaz replied:
Ok, thanks.

That looks much more encapsulated right now.

earnubs commented:
@bartaz have another look, addressing your comment by using a ref to the component as the target of the clipboard lib rather than a selector, and cleaning up on unmount.
